src/shared/dlt_common.c

```diff
@@ -3992,7 +4006,7 @@ void dlt_check_envvar()
         int mode = 0;
 
         if (sscanf(env_log_mode, "%d", &mode))
-            dlt_log_init(mode);
+            dlt_log_init_p2(mode);
```

Comment:
I try to build with your change, but it throws error:
[ 40%] Linking C executable dlt-convert
../lib/libdlt.so.2.18.8: undefined reference to `dlt_log_init'
collect2: error: ld returned 1 exit status
src/console/CMakeFiles/dlt-convert.dir/build.make:96: recipe for target 'src/console/dlt-convert' failed
make[2]: *** [src/console/dlt-convert] Error 1
CMakeFiles/Makefile2:295: recipe for target 'src/console/CMakeFiles/dlt-convert.dir/all' failed
make[1]: *** [src/console/CMakeFiles/dlt-convert.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

I check and it turns out that this API dlt_check_envvar() is also used by dlt-user, so I suspect this change somehow affects other components.
The second thing is that as you could see from the error, it seems that there is something wrong in the source code when we are trying to change the API like that.
Please kindly check!
Regards